ArtOfIllusion/src/artofillusion/ObjectTextureDialog.java

```diff
@@ -145,15 +144,7 @@ public ObjectTextureDialog(LayoutWindow parent, ObjectInfo objects[], boolean in
     
     for (Texture texture : PluginRegistry.getPlugins(Texture.class))
     {
-      try
```

Comment:
Noticed that you have removed a lot of try blocks in this PR. Some of them certainly look as if they are not guarding anything, but are you _sure_ of that? In particular, what happens if you open a file that contains a texture that relies on a plugin, and you do not have the plugin?

@peastman would it even be safe to edit and save the file in such a case? I looked at the scene loading code, and do not see where unreadable texture data would be retained?